utils/crun/patches/010-gcc15.patch

```diff
@@ -0,0 +1,11 @@
+--- a/tests/tests_libcrun_fuzzer.c
```

Comment:
https://github.com/containers/crun/commit/9acf13d6c1e9a76353bf7f83025b84a9bcf13a53.patch

maybe use upstream patch instead?

Reply:
IIRC, that fails with GCC14.

Reply:
I could be wrong but I think the issue with

```
sig_chld (int)
```

is that this is valid in C++ and C23 but not prior versions. So technically GCC14 would work with an explicit std=c23.